src/main/java/lesser/citibike/CitiBikeHelper.java

```diff
@@ -0,0 +1,98 @@
+package lesser.citibike;
+
+import io.reactivex.rxjava3.core.Single;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CitiBikeHelper {
```

Comment:
This class doesn't actually help. You are making the network calls multiple times when you should only be making the calls once.